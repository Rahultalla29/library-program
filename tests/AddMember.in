ADD MEMBER Jeff
ADD MEMBER Chris Jones
ADD MEMBER Jeff Chris Jones Jr.
MEMBER 100000
MEMBER 100001
MEMBER 100002
add member jacob
member 100003
EXIT
