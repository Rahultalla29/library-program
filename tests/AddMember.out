user: Success.

user: Success.

user: Success.

user: 100000: Jeff

user: 100001: Chris Jones

user: 100002: Jeff Chris Jones Jr.

user: Success.

user: 100003: jacob

user: Ending Library process.
